ArcGISRuntimeSDKQt_QMLSamples/Analysis/TraceUtilityNetwork/TraceUtilityNetwork.qml

```diff
@@ -214,7 +237,20 @@ Rectangle {
             lineLayer.selectFeaturesWithQuery(lineParams, Enums.SelectionModeAdd);
         }
 
-        onComponentCompleted: load();
+        onErrorChanged: {
+            dialogText.text = qsTr("%1 - %2".arg(error.message).arg(error.additionalMessage));
+            traceCompletedDialog.open();
+        }
+
+        onComponentCompleted: {
```

Comment:
does this work? I've noticed in some cases that `onComponentCompleted` doesn't work but `Component.onCompleted` does. No preference on my end, just making sure it works properly

Reply:
Yes it does work. 👍